www/app.py

```diff
@@ -52,6 +55,49 @@ def __init__(self, name, pin, rising_event, falling_event):
         self.falling_event = falling_event
         self.old_val = False
 
+def get_local_ip():
+    s = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
+    s.connect(("8.8.8.8", 80))
+    ip = s.getsockname()[0]
+    s.close()
+    return ip
+
+def register_with_web():
+    global web_id
+    global payload
+    payload = {'name': 'Chipy', 'owner': 'Mr. Hurlburt', 'local_ip': get_local_ip()}
+    print "Registering with rovercode-web"
+    r = requests.post("https://rovercode.com/mission-control/rovers/", payload)
```

Comment:
I would suggest moving the base URL somewhere that can be accessed by everything needing it so you only will have to change it in one place in the future.